tests/integration/integration_test.py

```diff
@@ -952,6 +750,164 @@ def my_on_revoke(consumer, partitions):
     c.close()
 
 
+def verify_avro():
+    base_conf = {'bootstrap.servers': bootstrap_servers,
+                 'error_cb': error_cb,
+                 'api.version.request': api_version_request,
+                 'api.version.fallback.ms': 0,
+                 'broker.version.fallback': '0.11.0.0',
+                 'schema.registry.url': schema_registry_url,
+                 'default.topic.config': {
+                     'produce.offset.report': True
+                 }}
+
+    consumer_conf = dict(base_conf, **{
+        'group.id': 'test.py',
+        'session.timeout.ms': 6000,
+        'enable.auto.commit': False,
+        'on_commit': print_commit_result,
+        'default.topic.config': {
+            'auto.offset.reset': 'earliest'
+        }})
+
+    run_avro_loop(base_conf, consumer_conf)
+
+
+def verify_avro_https(mode_conf):
+    if mode_conf is None:
+        abort_on_missing_configuration('avro-https')
+
+    base_conf = dict({'bootstrap.servers': bootstrap_servers,
+                      'error_cb': error_cb,
+                      'api.version.request': api_version_request},
+                     **mode_conf)
```

Comment:
verify_avro() did it the other way around: base_conf, **{..}. Is there a reason this is reversed here?

Reply:
doesn't matter as long as you unpack the second dict. I'll flip it for consistency.